src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/NocClaimantHelper.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.et.common.model.bulk.types.DynamicFixedListType;
+import uk.gov.hmcts.et.common.model.bulk.types.DynamicValueType;
+import uk.gov.hmcts.et.common.model.ccd.types.ChangeOrganisationRequest;
+import uk.gov.hmcts.et.common.model.ccd.types.Organisation;
+import uk.gov.hmcts.ethos.replacement.docmosis.domain.ClaimantSolicitorRole;
+
+import java.time.LocalDateTime;
+import static uk.gov.hmcts.et.common.model.ccd.types.ChangeOrganisationApprovalStatus.APPROVED;
+
+@Component
+public class NocClaimantHelper {
+    public ChangeOrganisationRequest createChangeRequest(Organisation newOrganisation,
+                                                         Organisation oldOrganisation) {
+        DynamicFixedListType roleItem = new DynamicFixedListType();
+        DynamicValueType dynamicValueType = new DynamicValueType();
+        dynamicValueType.setCode(ClaimantSolicitorRole.CLAIMANTSOLICITOR.getCaseRoleLabel());
+        dynamicValueType.setLabel(ClaimantSolicitorRole.CLAIMANTSOLICITOR.getCaseRoleLabel());
+        roleItem.setValue(dynamicValueType);
```

Comment:
Minor point - can be a simplified a little with one of the methods in DynamicFixedListType
```suggestion
        DynamicFixedListType roleItem =
                new DynamicFixedListType(ClaimantSolicitorRole.CLAIMANTSOLICITOR.getCaseRoleLabel());
```